src/services/dashboards/DashboardsLNB.vue

```diff
@@ -0,0 +1,138 @@
+                {
+                    type: 'item', id: 'Menu Item', label: 'Menu Item', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'work_01' } },
+                },
+                {
+                    type: 'item', id: 'Budget Summary', label: 'Budget Summary', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'work_02' } },
+                },
+            ])),
+            projectMenuSet: computed<LNBMenu[]>(() => (
+                [
+                    [{
+                        type: 'title',
+                        label: 'Project_01',
+                        id: 'Project_01',
+                        foldable: true,
+                    },
+                    {
+                        type: 'item', id: 'Project_01_Dashboard', label: 'Project_01_Dashboard', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'project_01_1' } },
+                    },
+                    {
+                        type: 'item', id: 'Project_01_Dashboard2', label: 'Project_01_Dashboard2', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'project_01_2' } },
+                    }],
+                    [{
+                        type: 'title',
+                        label: 'Project_02',
+                        id: 'Project_02',
+                        foldable: true,
+                    },
+                    {
+                        type: 'item', id: 'Project_02_Dashboard', label: 'Project_02_Dashboard', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'project_02_1' } },
+                    },
+                    {
+                        type: 'item', id: 'Project_02_Dashboard2', label: 'Project_02_Dashboard2', to: { name: DASHBOARDS_ROUTE.DETAIL._NAME, params: { dashboardId: 'project_02_2' } },
+                    }],
+                ]
+            )),
+            menuSet: computed<LNBMenu[]>(() => [
+                {
+                    type: 'item',
+                    label: 'View All Dashboards', // song-lang
+                    id: MENU_ID.DASHBOARDS,
+                    foldable: false,
+                    to: { name: DASHBOARDS_ROUTE.ALL._NAME },
+                },
+                { type: 'divider' },
+                { type: 'favorite-only' },
+                { type: 'top-title', label: 'Workspace' },
+                ...filterFavoriteItems(state.workSpaceMenuSet),
+                { type: 'top-title', label: 'Project' },
+                ...filterFavoriteItems(state.projectMenuSet),
+            ]),
+        });
+
+        const filterFavoriteItems = (menuItems: LNBItem[] = []) => {
+            if (!state.favoriteOnly) return menuItems;
+            const result = [] as LNBItem[];
+            menuItems.forEach((d) => {
+                if (d.id && state.favoriteItemMap[d.id] || d.type !== MENU_ITEM_TYPE.ITEM) {
+                    result.push(d);
+                }
+            });
+            return result;
```

Comment:
```suggestion
           return menuItems.filter((d) => d.id && state.favoriteItemMap[d.id] || d.type !== MENU_ITEM_TYPE.ITEM)
```

이렇게 압축할 수 있을 것 같습니다.

+ if 구문에 서로 다른 논리연산자를 연달아 사용하여 한 번에 이해하기 조금 어령루 것 같습니다. 연관이 깊은 것을 한 번 더 괄호처리하여 가독성을 올리면 어떨까요?